From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@pcw3187__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-pawrig). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

GitHubber17 committed:
Merge pull request #33288 from ajlam/release-connect-orcas

Update MySQL and PG howto server parameters CLI and Portal

ajlam committed:
Merge pull request #2 from MicrosoftDocs/release-connect-orcas

Update fork

ajlam committed:
Update MySQL and Postgres howto firewall CLI and portal

ajlam committed:
Update MySQL howto configure SSL

ajlam committed:
Update manager

ajlam committed:
Update MySQL and PG sample scripts

ajlam committed:
Merge branch 'release-connect-orcas' of https://github.com/ajlam/azure-docs-pr into release-connect-orcas

ajlam committed:
Update MySQL pricing tier

ajlam committed:
Update PG pricing tier

rachel-msft committed:
updating limits, monitoring, quickstart

rmca14 committed:
Merge pull request #33327 from rachel-msft/monitor-limits-quickstart

updating limits, monitoring, quickstart

Lisaco88 committed:
Merge pull request #33322 from ajlam/release-connect-orcas

Update MySQL and PG firewall (CLI and portal), MySQL SSL, MySQL and PG scripts, pricing tiers

ajlam committed:
Merge pull request #3 from MicrosoftDocs/release-connect-orcas

Update fork

ajlam committed:
Update MySQL and PG connect and query

ajlam committed:
Update limits

ajlam committed:
Update innodb and connection limits

ajlam committed:
Update Mysql limits

ajlam committed:
update mysql firewall cli

ajlam committed:
Update mysql firewall portal

ajlam committed:
Update PG firewall cli

ajlam committed:
Update pg firewall portal

ajlam committed:
Update PG concepts servers

ajlam committed:
Fix images in firewall portal

Commit:
Merge pull request #33351 from ajlam/patch-4

Update PG concepts servers

ajlam committed:
Update bullet order

Lisaco88 committed:
Merge pull request #33340 from ajlam/release-connect-orcas

Update MySQL and PG connect and query

rmca14 committed:
Merge pull request #33341 from ajlam/patch-2

Update MySQL limits, revert to master MySQL and PG firewall CLI and portal

ajlam committed:
Merge pull request #4 from MicrosoftDocs/release-connect-orcas

Update fork

ajlam committed:
Fix typos in connect and query

rachel-msft committed:
updating portal restore pg, mysql

rachel-msft committed:
Merge branch 'release-connect-orcas' of https://github.com/MicrosoftDocs/azure-docs-pr into restore-portal

rachel-msft committed:
updating overview pg

rachel-msft committed:
updating overview pg

rachel-msft committed:
updating portal restore mysql

rachel-msft committed:
updating quickstarts mysql, postgres

ajlam committed:
Update MySQL howto create and manage server

rachel-msft committed:
updating tutorials mysql, postgres

GitHubber17 committed:
Merge pull request #33377 from rachel-msft/overview-pg

updating overview pg

v-alje committed:
Resolving merge conflict

v-alje committed:
Bringing even with master.

ktoliver committed:
Merge pull request #33375 from rachel-msft/restore-portal

updating portal restore pg, mysql

ajlam committed:
Update screenshots

Commit:
Updated howto-manage-vnet-using-cli.md to clarify prerequisites and changed the Important section location to be infront of the sample scripts.

ktoliver committed:
Merge pull request #33385 from ajlam/release-connect-orcas

 Update MySQL howto create and manage server, fix typos in connect & query

v-alje committed:
Bringing even with master.

ajlam committed:
Merge pull request #5 from MicrosoftDocs/release-connect-orcas

Update fork

ajlam committed:
Updates from doc bash

rachel-msft committed:
updating tutorials mysql, postgres

rachel-msft committed:
Merge branch 'release-connect-orcas' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorials

ajlam committed:
Update PG backup and pricing tiers

rachel-msft committed:
Merge branch 'release-connect-orcas' of https://github.com/MicrosoftDocs/azure-docs-pr into quickstarts

rachel-msft committed:
updating quickstarts mysql, postgres

paulathurman committed:
edit pass: tutorial-bikeshare-dataprep

GitHubber17 committed:
Merge pull request #33423 from ajlam/release-connect-orcas

Doc bash changes

rachel-msft committed:
updating tutorial mysql

rachel-msft committed:
Merge branch 'release-connect-orcas' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorials

Lisaco88 committed:
Fixed ms.date

Lisaco88 committed:
Update tutorial-design-database-using-cli.md

Lisaco88 committed:
Merge pull request #33384 from rachel-msft/quickstarts

updating quickstarts mysql, postgres

Lisaco88 committed:
Merge pull request #33387 from rachel-msft/tutorials

updating tutorials mysql, postgres

ajlam committed:
Merge pull request #6 from MicrosoftDocs/release-connect-orcas

Update fork

v-alje committed:
Bringing even with master.

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-bikeshare-dataprep

Commit:
Merge branch 'release-connect-orcas' of https://github.com/MicrosoftDocs/azure-docs-pr into localbranch

Daily pull from master

Commit:
Fixed broken links to scripts.

Commit:
Fixed highlighted lines in each script

GitHubber17 committed:
Merge pull request #33396 from bolzmj/release-connect-orcas

Updated howto-manage-vnet-using-cli.md to clarify prerequisites and c…

ajlam committed:
Updates from doc bash continued

v-alje committed:
Bringing even with master.

ajlam committed:
Update toc

rachel-msft committed:
remove private articles

paulathurman committed:
edit pass: tutorial-bikeshare-dataprep

GitHubber17 committed:
Merge pull request #33521 from rachel-msft/remove-articles

Remove private articles

itechedit committed:
Update TOC.yml

itechedit committed:
Update TOC.yml

itechedit committed:
Merge pull request #33518 from ajlam/release-connect-orcas

Updates from doc bash #2

ajlam committed:
Merge pull request #7 from MicrosoftDocs/release-connect-orcas

update fork

Commit:
Removed all VNet articles, medial and TOC entries for Postgresql and MySQL. We are shipping release-connect-orcas on or around 2/20/2018. VNet is still in private preview and will ship at a later date.

ajlam committed:
Fix max connection typos

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-bikeshare-dataprep

PRMerger12 committed:
Merge pull request #33558 from ajlam/release-connect-orcas

Fix typo in max connections

ajlam committed:
Merge pull request #8 from MicrosoftDocs/release-connect-orcas

update fork

Lisaco88 committed:
Merge pull request #33555 from bolzmj/release-connect-orcas

Removed all VNet articles, medial and TOC entries for Postgresql and …

v-alje committed:
Resolving merge conflicts.

paulathurman committed:
edit pass: tutorial-bikeshare-dataprep

v-alje committed:
Bringing even with master.

v-viksur committed:
(ForumsTeam) Broken Link 

In this documentation underr the section "Leasing a Page Blob" "code samples" Hyperlink has broken. I am requesting you to update the broken link

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-bikeshare-dataprep

paulathurman committed:
edit pass: tutorial-bikeshare-dataprep

paulathurman committed:
edit pass: tutorial-bikeshare-dataprep

Lisaco88 committed:
edit pass: active-directory-app-gallery-listing

v-alje committed:
Bringing even with master.

ajlam committed:
Merge pull request #9 from MicrosoftDocs/release-connect-orcas

Update fork

v-alje committed:
Bringing even with master.

jan-eng committed:
fixed some language in biz continuity articles and pricing tier page

v-alje committed:
Bringing even with master.

jan-eng committed:
fixed some language in biz continuity articles and pricing tier page

Ja-Dunn committed:
Merge pull request #33719 from jan-eng/release-connect-orcas

fixed some language in biz continuity articles and pricing tier page

ajlam committed:
Fix typo in MySQL CLI commands

PRMerger20 committed:
Merge pull request #33759 from ajlam/release-connect-orcas

 Fix typo in MySQL CLI commands

ajlam committed:
Fix typo in PostgreSQL sample script description

PRMerger19 committed:
Merge pull request #33760 from ajlam/release-connect-orcas

Fix typo in PostgreSQL sample script description

ajlam committed:
Merge pull request #10 from MicrosoftDocs/release-connect-orcas

Update fork

v-alje committed:
Bringing even with master.

v-geberr committed:
batch testing

Commit:
Added file ARM template topic

JasonWHowell committed:
Fixing small issues

JasonWHowell committed:
Fixing small issues

v-alje committed:
Bringing even with master.

Commit:
added TOC entry for ARM topic

Commit:
auto provisioning -> auto-provisioning

JasonWHowell committed:
Postgresql metadata updates

GitHubber17 committed:
Merge pull request #33878 from JasonWHowell/release-connect-orcas

Release connect orcas

JasonWHowell committed:
metdata updates

JasonWHowell committed:
Updating metadata for MySQL articles

JasonWHowell committed:
Updating metadata for MySQL articles

JasonWHowell committed:
resolving merge conflict

ktoliver committed:
Merge pull request #33935 from JasonWHowell/orcas-metadata

Postgresql metadata updates

JasonWHowell committed:
Updating metadata on PostgreSQL

JasonWHowell committed:
Updating metadata on PostgreSQL

JasonWHowell committed:
Fixing conflict

JasonWHowell committed:
Removing Microsoft Docs from titles

JasonWHowell committed:
Removing word topic from description

ktoliver committed:
Merge pull request #33943 from JasonWHowell/mysql-metadata

MySQL: Updating titles, authors, manager, and date metadata in all articles

Ja-Dunn committed:
Merge pull request #33953 from JasonWHowell/postgresql-metadata

PostgreSQL: Updating titles, authors, manager, and date metadata in all articles

Commit:
Update service-fabric-get-started-containers.md

JasonWHowell committed:
Update howto-restore-server-cli.md

JasonWHowell committed:
Update howto-restore-server-cli.md

Commit:
Merge pull request #33959 from JasonWHowell/patch-7

Update howto-restore-server-cli.md

Commit:
Merge pull request #33960 from JasonWHowell/patch-8

Update howto-restore-server-cli.md

JasonWHowell committed:
fixing server name and resource group name

JasonWHowell committed:
fixing conflict

PRMerger19 committed:
Merge pull request #33961 from JasonWHowell/release-connect-orcas

fixing server name and resource group name

JasonWHowell committed:
Update howto-manage-firewall-using-cli.md

PRMerger10 committed:
Merge pull request #33962 from JasonWHowell/patch-1

Update howto-manage-firewall-using-cli.md

Commit:
Update service-fabric-get-started-containers.md

linda33wj committed:
Add SAP ECC connector support

linda33wj committed:
Add ECC data type

dlepow committed:
Batch: update node.js ref links

sethmanheim committed:
Updating quotas tables

sethmanheim committed:
Updating and adding metadata

Commit:
added azure CLI link

v-alje committed:
Resolving merge conflict

paulathurman committed:
Merge branch 'master' into tutorial-bikeshare-dataprep

Commit:
added links and image

Commit:
yanked hub name property from linked IoT Hub object in template -- it's not needed

v-geberr committed:
wip

dlepow committed:
Fixed TOC link

v-geberr committed:
wip

Commit:
added feedback. Cleaned up some acrolynx errors

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dps-quick-setup-auto-provision-arm
sync with upstream

dlepow committed:
API link updates

paulathurman committed:
Update tutorial-bikeshare-dataprep.md

v-alje committed:
Bringing even with master.

v-geberr committed:
wip

billmath committed:
updating GAL support

v-geberr committed:
remove entities for now

v-geberr committed:
create app

v-geberr committed:
collaborator

linda33wj committed:
Refine SAP ECC connector

v-geberr committed:
links between owner and collaborator defintions

sethmanheim committed:
Removing concurrent connection limit

alexwun committed:
Link to shared sfpkg doc

Lisaco88 committed:
Changed ms.date to 2018

PRMerger14 committed:
Merge pull request #33602 from v-viksur/patch-22

(ForumsTeam) Broken Link

Lisaco88 committed:
Updated ms.date to 2018

cherylmc committed:
deleteconnection

PRMerger17 committed:
Merge pull request #34101 from alexwun/master

Link to shared sfpkg doc

Lisaco88 committed:
Merge pull request #34096 from v-geberr/0227-collaborator

collaborator

sanjeev3 committed:
Update sdk-and-samples.md

Lisaco88 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into active-directory-app-gallery-listing

cherylmc committed:
format

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into node

PRMerger19 committed:
Merge pull request #34102 from cherylmc/er2

deleteconnection

cherylmc committed:
format

PRMerger14 committed:
Merge pull request #34104 from sanjeev3/master

Update sdk-and-samples.md - added a link.

v-geberr committed:
create-batch

paulathurman committed:
Update tutorial-bikeshare-dataprep.md

dlepow committed:
fixed link

cherylmc committed:
format

kashimiz committed:
Added Azure Functions local host logs location.

bandersmsft committed:
cost mgt - updating tutorials for MVC guidance changes

Lisaco88 committed:
Merge pull request #34030 from sethmanheim/quotas-fix

Updating quotas tables

bandersmsft committed:
updated screenshot

malopMSFT committed:
Update on security overview for LKM rule

Describing platform behavior under special conditions

v-geberr committed:
fix link

malopMSFT committed:
Update security-overview.md

linda33wj committed:
Address comments

cherylmc committed:
format

sethmanheim committed:
Removing incorrect info

ecfan committed:
Add missing redirection

Commit:
freshness

Commit:
Merge pull request #34086 from billmath/aadc

updating GAL support

v-alje committed:
Bringing even with master.

bandersmsft committed:
fixing yaml error

Lisaco88 committed:
Merge pull request #33934 from JimacoMS2/dps-quick-setup-auto-provision-arm

DPS auto-provisioning Quickstart for ARM template

PRMerger12 committed:
Merge pull request #34112 from sethmanheim/eh-rev

Removing incorrect info

Commit:
forgot to change date

rwike77 committed:
Updating

dlepow committed:
standardized API ref links

oslake committed:
Update sql-database-service-tiers-table-elastic-pools.md

v-shils committed:
Merge pull request #34017 from linda33wj/master

Add SAP ECC connector support

iainfoulds committed:
[VM] Fixes MicrosoftDocs/azure-docs#5056

Adding specific Ubuntu 16.04 LTS version to the doc to address customer feedback on other distros appearing different.

oslake committed:
Update sql-database-resource-limits.md

v-geberr committed:
pre acro

Lisaco88 committed:
Merge pull request #34108 from bandersmsft/master

Cost Mgt - Updates for changes to MVC tutorial guidance

v-shils committed:
Merge pull request #34105 from cherylmc/er3

format

v-geberr committed:
acro

PRMerger20 committed:
Merge pull request #34118 from rwike77/patch-31

Updating

Lisaco88 committed:
Merge pull request #34113 from Blackmist/freshness

freshness

PRMerger20 committed:
Merge pull request #34121 from iainfoulds/patch-13

[VM] Fixes MicrosoftDocs/azure-docs#5056

v-shils committed:
Merge pull request #34115 from ecfan/fix-redirection

Add missing redirection

ggailey777 committed:
Fixes MicrosoftDocs/azure-docs#5060

PRMerger15 committed:
Merge pull request #34119 from oslake/patch-1

Update sql-database-service-tiers-table-elastic-pools.md

rwike77 committed:
updated container quickstart

Commit:
freshness

Commit:
version number

Lisaco88 committed:
Merge pull request #34024 from dlepow/node

Batch: update node.js ref links

PRMerger14 committed:
Merge pull request #34127 from rwike77/freshness

updated container quickstart

amanarneja committed:
(ForumsTeam) Fixing minor Typo

the D in GDPR should be Data and not Date

v-geberr committed:
final edits

amanarneja committed:
(ForumsTeam) Fix for MicrosoftDocs/azure-docs#5068

Lisaco88 committed:
edit pass: active-directory-app-gallery-listing

PRMerger14 committed:
Merge pull request #34126 from Blackmist/freshness

freshness

Commit:
Merge pull request #34129 from amanarneja/patch-1

(ForumsTeam) Fixes MicrosoftDocs/azure-docs#5065

v-geberr committed:
edit

PRMerger12 committed:
Merge pull request #34131 from amanarneja/patch-2

(ForumsTeam) Fix for MicrosoftDocs/azure-docs#5068

Commit:
Merge pull request #34124 from ggailey777/patch-10

Link fixes in Service Bus docs MicrosoftDocs/azure-docs#5060

ShawnJackson committed:
Merge pull request #33619 from paulathurman/tutorial-bikeshare-dataprep

edit pass: Tutorial bikeshare dataprep

ShawnJackson committed:
Update active-directory-app-gallery-listing.md

Commit:
freshness

barbkess committed:
remove index-all-articles

ShawnJackson committed:
Merge pull request #33620 from Lisaco88/active-directory-app-gallery-listing

edit pass: active-directory-app-gallery-listing

PRMerger13 committed:
Merge pull request #34134 from Blackmist/freshness

freshness

v-geberr committed:
remove git placeholder

PRMerger14 committed:
Merge pull request #34109 from malopMSFT/patch-114

Update on security overview for LKM rule

ajlam committed:
Merge pull request #11 from MicrosoftDocs/release-connect-orcas

Update fork

ajlam committed:
Add Azure CLI extension

Ja-Dunn committed:
Merge pull request #33860 from v-geberr/0222-tut-batch-testing

batch testing

Brenduns committed:
jeffs updates for marketplace

Ja-Dunn committed:
Merge pull request #34107 from kashimiz/functions-local-host-logs

Added Azure Functions local host logs location.

PRMerger15 committed:
Merge pull request #33958 from tugup/patch-2

Update service-fabric-get-started-containers.md

ajlam committed:
Update pricing tier screenshots

PRMerger18 committed:
Merge pull request #34138 from Brenduns/jeffs-updates

jeffs updates for marketplace

ajlam committed:
Update portal screenshots

rjagiewich committed:
Merge pull request #34135 from barbkess/toc-updates

remove index-all-articles

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #34139 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #34137 from ajlam/release-connect-orcas

Add Azure CLI extension & pricing tier screenshots

v-alje committed:
Merge pull request #34122 from MicrosoftDocs/release-connect-orcas

All pull requests have already been validated.
